src/java/fr/paris/lutece/plugins/appointment/web/AppointmentApp.java

```diff
@@ -473,9 +471,10 @@ public synchronized XPage getViewAppointmentCalendar( HttpServletRequest request
                             .get( ).getDate( );
                 }
             }
-            if ( firstDateOfFreeOpenSlot == null )
+            if ( firstDateOfFreeOpenSlot == null && formMessages != null
```

Comment:
I apologize for not seeing your comment earlier. It appears that I was not receiving notifications from GitHub. I will  fix my notification settings so that I can respond to feedback more promptly in the future. Thank you for bringing this to my attention.